src/pages/GenericErrorPage/index.js

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {View} from 'react-native';
+import Icon from '../../components/Icon';
+import defaultTheme from '../../styles/themes/default';
+import * as Expensicons from '../../components/Icon/Expensicons';
+import Text from '../../components/Text';
+import Button from '../../components/Button';
+import LogoWordmark from '../../../assets/images/expensify-wordmark.svg';
+import withLocalize, {withLocalizePropTypes} from '../../components/withLocalize';
+import withWindowDimensions, {windowDimensionsPropTypes} from '../../components/withWindowDimensions';
+import compose from '../../libs/compose';
+import * as Session from '../../libs/actions/Session';
+import variables from '../../styles/variables';
+import styles from '../../styles/styles';
+import ErrorBodyText from './ErrorBodyText';
+import TextLink from '../../components/TextLink';
+import CONST from '../../CONST';
+
+const propTypes = {
+    ...withLocalizePropTypes,
+    ...windowDimensionsPropTypes,
+
+    /** Function call on Refresh button click */
+    onRefresh: PropTypes.func.isRequired,
+};
+
+const genericErrorPage = (props) => {
```

Comment:
```suggestion
const GenericErrorPage = (props) => {
```
component starts with Capitals.